src/resources/common/Configuration.ts

```diff
@@ -117,6 +121,7 @@ export interface ConfigurationItem {
     securityConfiguration: SecurityConfiguration;
     subscriptionConfiguration: SubscriptionsConfiguration;
     userTransactionsConfiguration?: UserTransactionsConfiguration;
+    widgetConfiguration?: WidgetConfiguration | null;
```

Comment:
sorry for the confusion, but I went with `checkoutConfiguration` as the name here!